packages/analytics-js/src/components/userSessionManager/UserSessionManager.ts

```diff
@@ -262,6 +267,26 @@ class UserSessionManager implements IUserSessionManager {
     }
   }
 
+  /**
+   * A function to make an external request to set the cookie from server side
+   * @param key       cookie name
+   * @param value     encrypted cookie value
+   */
+  setServerSideCookie(key: string, value: string): void {
+    let baseUrl = state.lifecycle.activeDataplaneUrl.value;
+    if (typeof state.loadOptions.value.cookieServerUrl === 'string') {
```

Comment:
Let's also validate that it's a valid URL.

Reply:
Addressed